src/collar/oc_core.lsl

```diff
@@ -499,7 +499,8 @@ state active
                         // do weld
                         llMessageLinked(LINK_SET, NOTIFY, "1Please wait...", g_kWelder);
                         llMessageLinked(LINK_SET, NOTIFY_OWNERS, "%WEARERNAME%'s collar has been welded", g_kWelder);
-                        llMessageLinked(LINK_SET, LM_SETTING_SAVE, "intern_weld=1", g_kWelder);
+                        llMessageLinked(LINK_SET, LM_SETTING_SAVE, "intern_weld=1", "origin");
```

Comment:
This is not necessary and breaks the weld function since oc_settings will automatically set weldby based upon the key value